src/character.h

```diff
@@ -2317,6 +2323,8 @@ class Character : public Creature, public visitable<Character>
         // --------------- Values ---------------
         pimpl<SkillLevelMap> _skills;
 
+        std::set<proficiency_id> _proficiencies;
```

Comment:
I think this would be better as a `cata::flat_set` (generally better for small sets where reference stability is not required).  I don't think this one is performance-critical, though, so not super important.

Reply:
I'll defer on that, if you don't mind, simply to not make as many changes. I don't expect this to stay as a set for too long, though, because it'll have to change when adding the ability to learn proficiencies.